recipes/natural_language_processing/chatbot/Makefile

```diff
@@ -13,6 +13,12 @@ build:
 .PHONY: bootc
 bootc: quadlet
 	podman build $${FROM:+--from $${FROM}} --cap-add SYS_ADMIN --build-arg "SSHPUBKEY=$(SSHPUBKEY)" -f bootc/Containerfile -t ${BOOTCIMAGE} .
+	@echo ""
+	@echo "Successfully built bootc image '${BOOTCIMAGE}'."
```

Comment:
Should we also mention pushing to a registry and then installing via bootc install?

Reply:
Outside of my immediate scope but I am happy to add if you have text proposal.  Otherwise, we could move that into the README.

Reply:
@cgwalters WDYT?